05-workshop_functions.Rmd

```diff
@@ -92,7 +92,8 @@ Or if the package hasn't been loaded using the package::function() format:
 
 # Cheat sheets
 
-Popular packages have cheat sheets and R Studio's can be found [here](https://rviews.rstudio.com/2021/03/10/rstudio-open-source-resorurces/) which download as a pdf.
+Popular packages have 
+[cheat sheets and R Studio's](https://rviews.rstudio.com/2021/03/10/rstudio-open-source-resorurces/) which download as a pdf.
```

Comment:
Might be better to remove "R Studio's" to just read [cheat sheets](https://rviews.rstudio.com/2021/03/10/rstudio-open-source-resorurces/) which are available as pdfs.